community/settings.py

```diff
@@ -132,16 +131,16 @@
         'default': db_from_env
     }
     CACHES = {
-        'default': {
-            'BACKEND': 'django_bmemcached.memcached.BMemcached',
-            'LOCATION': os.environ.get('MEMCACHEDCLOUD_SERVERS').split(','),
-            'OPTIONS': {
-                'username': os.environ.get('MEMCACHEDCLOUD_USERNAME'),
-                'password': os.environ.get('MEMCACHEDCLOUD_PASSWORD')
-            }
+        "default": {
+            "BACKEND": "django_redis.cache.RedisCache",
```

Comment:
This is awesome. I'm excited to use Redis for caching ![bitmoji](https://render.bitstrips.com/v2/cpanel/9666023-183380029_1-s1-v1.png?transparent=1&width=246)